ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -59,41 +61,41 @@ internal class PaywallViewModelImpl(
     preview: Boolean = false,
 ) : ViewModel(), PaywallViewModel {
 
-    private val variableDataProvider = VariableDataProvider(applicationContext, preview)
```

Comment:
Hmm any reason we removed this `private`?

Reply:
Hmm good catch, it was because `calculateState` was defined outside but I put it inside now.